scripts/sct_get_centerline.py

```diff
@@ -10,13 +10,16 @@
 from msct_image import Image
 from sct_image import orientation
 from msct_types import Centerline
-from sct_viewer import ClickViewerPropseg
+
 from sct_straighten_spinalcord import smooth_centerline
 
 
```

Comment:
add a `get_centerline()` function and move the above imports inside, then split the main() function and move everything after the parser operation to that new function. And it will be done.